hashintel/hash
# Improve ranking of users in @ mention menu

## 🌟 What is the purpose of this PR?



~Currently the Mention Suggester (or @ menu) includes all users which isn't a desired behaviour. This PR updates it to only display members of an org if the user is in an org's workspace or just the user if the user is in their own workspace.~
This PR updates the Mention Suggester to display members of the active/current workspace first before other users.


## 🔗 Related links





- [Asana task](https://app.asana.com/0/0/1202510248434796) _(internal)_


## ❓ How to test this?



1.  Checkout the branch
1.  Go to the org's workspace and confirm the org members show up first in the mention suggester (or @ menu)



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202510248434796
